storage/scratchpad/Cargo.toml

```diff
@@ -25,16 +25,16 @@ proptest = { workspace = true, optional = true }
 rayon = { workspace = true }
 thiserror = { workspace = true }
 
-[target.'cfg(unix)'.dependencies]
-jemallocator = { workspace = true }
-
 [dev-dependencies]
 aptos-types = { workspace = true, features = ["fuzzing"] }
 bitvec = { workspace = true }
 once_cell = { workspace = true }
 proptest = { workspace = true }
 rand = { workspace = true }
 
+[target.'cfg(unix)'.dependencies]
```

Comment:
did you mean `[target.'cfg(unix)'.dev-dependencies]`? (in the current form it's just a reorder?)

Reply:
Yeah, let's put dev-dependencies

Reply:
fixed.